charts/jenkins/templates/jenkins-controller-ingress.yaml

```diff
@@ -42,11 +42,11 @@ spec:
           serviceName: {{ template "jenkins.fullname" . }}
           servicePort: {{ .Values.controller.servicePort }}
 {{ end }}
-{{- else }}
-{{ tpl (toYaml .Values.controller.ingress.paths | indent 6) . }}
 {{- if .Values.controller.ingress.path }}
         path: {{ .Values.controller.ingress.path }}
 {{- end -}}
+{{- else }}
+{{ tpl (toYaml .Values.controller.ingress.paths | indent 6) . }}
```

Comment:
can you add a test for this?
https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/tests/jenkins-controller-ingress-test.yaml

Reply:
Done. Not sure if I have done it correctly though.